test/functional/page_objects/time_picker.js

```diff
@@ -264,6 +264,22 @@ export function TimePickerPageProvider({ getService, getPageObjects }) {
 
       await this.closeQuickSelectTimeMenu();
     }
+
+    async setHistoricalDataRange() {
+      await this.setDefaultAbsoluteRange();
+    }
+
+    async setDefaultDataRange() {
+      const fromTime = 'Jan 1, 2018 @ 00:00:00.000';
+      const toTime = 'Apr 13, 2018 @ 00:00:00.000';
+      await this.setAbsoluteRange(fromTime, toTime);
+    }
+
+    async setLogstashDataRange() {
+      const fromTime = 'Apr 9, 2018 @ 00:00:00.000';
+      const toTime = 'Apr 13, 2018 @ 00:00:00.000';
```

Comment:
Why these dates in this fn, and the previous fn?

Reply:
It was transferred from `dashboard_page.js` because it is not dashboard action, but time picker. There are 3 of these functions and they set an interval according to data in ES archivers.